### PR TITLE
Mark as unread

### PR DESCRIPTION
**Pull request by bago2k4**
### Review after #16 

Main PR: open-company/open-company-web#856

To test backward compatibility:
- clean all your DBs
- onboard a new org/user
- add some posts
- trigger an email digest
- trigger an slack digest
- do you see both digest?

### CODE REVIEW

**Comment by bago2k4** [comment]
[Ability to mark a post as unread](https://trello.com/c/KRA9KvKU/1997-ability-to-mark-a-post-as-unread)

**Comment by belucid** [comment]
@bago2k4 this one should be backward compatible with current Web UI so changed the target to `mainline`.

File: src/oc/change/api/change.clj [hunk 1]
```diff
@@ -34,6 +43,13 @@
...
 
+  :delete! (fn [ctx] (let [user (:user ctx)
+                           user-id (:user-id user)
+                           delete-item (read/delete! post-uuid user-id)]
+                       (if (nil? delete-item)
+                         {:deleted-items :ok}
+                         {:deleted-items false})))
+
   ;; Responses
   :handle-ok (fn [ctx] (render-count (:existing-post ctx))))
```
Comment (cjsauer) [inline src/oc/change/api/change.clj]: @bago2k4 it looks like `render-count` is a legacy name that should be updated. The function does not appear to return a "count" anymore, and instead returns an entire read/seen map. Maybe `render-post-visibility`, or `render-post-change`?

File: src/oc/change/api/change.clj [hunk 2]
```diff
@@ -34,6 +43,13 @@
                                                :read read-data}}
                         false))
 
+  :delete! (fn [ctx] (let [user (:user ctx)
+                           user-id (:user-id user)
+                           delete-item (read/delete! post-uuid user-id)]
+                       (if (nil? delete-item)
+                         {:deleted-items :ok}
+                         {:deleted-items false})))
```
Comment (cjsauer) [inline src/oc/change/api/change.clj:51]: @bago2k4 I'm not totally familiar with Liberator, but what is the purpose of modifying the context map here? It doesn't appear that `:deleted-items` is actually used anywhere.

**Review by cjsauer — CHANGES_REQUESTED** [review]
@bago2k4 couple minor comments for you to review.

File: src/oc/change/api/change.clj (hunk 2 above)
Comment (bago2k4) [reply src/oc/change/api/change.clj:51]: nope @cjsauer is not directly used, but it's part of the liberator process to add to the context the results of all the DB operations.
If you look in the other liberator resources you should find more like this.

File: src/oc/change/api/change.clj (hunk 1 above)
Comment (bago2k4) [reply src/oc/change/api/change.clj]: done

**Comment by bago2k4** [comment]
@cjsauer addressed you comments, moved back to ready for review

**Review by cjsauer — APPROVED** [review]

**Pull request merged**